### PR TITLE
Avoided counting exceptions in AsyncClient docs.

### PR DESCRIPTION
**Pull request by felixxm**
Follow up to ad6bb20557f5c87de26aeb3afb061af942a8cc17.

### CODE REVIEW

File: docs/topics/testing/tools.txt [hunk 1]
```diff
@@ -2028,7 +2028,7 @@ or as ``self.async_client`` on any test.
 .. class:: AsyncClient(enforce_csrf_checks=False, raise_request_exception=True, *, headers=None, **defaults)
 
 ``AsyncClient`` has the same methods and signatures as the synchronous (normal)
-test client, with two exceptions:
+test client, with the following exceptions:
```
Comment (felixxm) [inline docs/topics/testing/tools.txt:2031]: Now we have three exceptions :smile:

**Review by jacobtylerwalls — APPROVED** [review]

**Review by nessita — APPROVED** [review]
LGTM

**Review by ngnpope — CHANGES_REQUESTED** [review]
LGTM, but fix `AsynClient` → `AsyncClient` in PR title / commit message.

**Comment by felixxm** [comment]
Thanks y'all :+1:

**Pull request merged**